managed/src/test/java/com/yugabyte/yw/controllers/UniverseControllerTest.java

```diff
@@ -2321,4 +2322,28 @@ public void testRemoveEnclosingDoubleQuotes() {
     // Null string
     assertEquals(null, UniverseController.removeEnclosingDoubleQuotes(null));
   }
+
+  @Test
+  public void testClearNodeDetails() {
```

Comment:
I think we have a kind of misunderstanding. Let me explain once again.
We need functions `testCreateDestroyServerTasks_<Some condition>`. In the first function - check the path when your code doesn't work (call the function and check that there were no invocations of clearNodeDetails).
In the second function - check the path when your code works (call the function and check that there were invocations of clearNodeDetails).

Reply:
@SergeyPotachev yeah , the code I have added here is in subtask. So finding it difficult to create the data for this subtask. And do we any examples where we have written test case for subtasks so that I can refer and add one here?

Reply:
To speedup the review I've prepared a prototype of the requested test.

[UniverseTaskBaseTest.java.txt](https://github.com/yugabyte/yugabyte-db/files/5914784/UniverseTaskBaseTest.java.txt)

Reply:
And remove changes from UniverseControllerTest.java, they are useless. Please.

Reply:
Added the testcases.